events/wc_undead_events.txt

```diff
@@ -56,29 +56,29 @@ province_event = {
 				}
```

Comment:
Still too big.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/80309583-020fb380-87e7-11ea-87b0-d60133aa0a56.png)

</details>

Reply:
In addition to lenght: The option B speaks of "he", though it's a she, isn't it? And what is "it" in the event text? "Yet there is still a matter of how much more loyal it can be."

Reply:
This is a followup to a previous event where a wicker golem is mentioned.